### PR TITLE
Show installed io plugins in ctapipe info, fixes #1540 

### LINKED ISSUE #1540: ctapipe-info should also list IO plugins
we should add an option to `ctapipe-info` to list installed EventSources

### PR DESCRIPTION
**Pull request by maxnoe**
fixes #1540

### CODE REVIEW

**Review by kosack — DISMISSED** [review]

File: ctapipe/tools/info.py [hunk 1]
```diff
@@ -199,5 +198,13 @@ def _info_system():
             print("{:>20.20s} -- {:<60.60s}".format(name, str(val)))
 
 
+def _info_plugins():
+    plugins = detect_and_import_io_plugins()
+    print("\n*** ctapipe io plugins ***\n")
+    for name in plugins:
+        version = get_module_version(name)
+        print(f"{name:>20s} -- {version}")
```
Comment (nbiederbeck) [inline ctapipe/tools/info.py:206]: Can we print a "no io plugins installed" if that is the case?

**Review by HealthyPear — CHANGES_REQUESTED** [review]
I also think we should be explicit if no I/O plugins are installed.

**Comment by maxnoe** [comment]
@HealthyPear @nbiederbeck done

**Comment by HealthyPear** [comment]
> @HealthyPear @nbiederbeck done

Sure you have pushed new modifications?
I don't see new commits

**Comment by maxnoe** [comment]
Sorry, the black commit hook complained and I didn't realized I hadn't commited.

**Review by nbiederbeck — APPROVED** [review]

**Review by HealthyPear — APPROVED** [review]

**Pull request merged**